Content.Server/Shuttles/Systems/DockingSystem.cs

```diff
@@ -363,13 +363,13 @@ public void Dock(DockingComponent dockA, DockingComponent dockB)
             if (TryComp(dockA.Owner, out DoorComponent? doorA))
             {
                 doorA.ChangeAirtight = false;
-                _doorSystem.StartOpening(doorA.Owner, doorA);
```

Comment:
im pretty sure these were `StartOpening` for a reason. furthermore, the code above bolts the airlocks before it starts opening them, so shouldnt this always fail if its checking now?

Reply:
Solved the bolt problem by moving the bolt step too after the TryOpen step. Seems to work properly in testing.